packages/circularprogress/src/index.d.ts

```diff
@@ -1,24 +1,39 @@
-import { ForwardRefExoticComponent, RefAttributes } from 'react'
+import { ForwardRefExoticComponent, HTMLAttributes, RefAttributes } from 'react'
 
-interface Styling {
-  css?: Record<string, string | number>
-  style?: Record<string, string | number>
-  className?: string
```

Comment:
these are react/dom native attrs and we shouldn't have to define them for all components. we can leverage the `HTMLAttributes` typing to get the list of this props. if we were to do this is our react source we wouldn't need the filterReactProps util anymore